lib/ours/errors.js

```diff
@@ -0,0 +1,378 @@
+'use strict'
+/*
+  This file is a reduced and adapted version of the main lib/internal/errors.js file defined at
+
+  https://github.com/nodejs/node/blob/master/lib/internal/errors.js
+
+  Don't try to replace with the original file and keep it up to date (starting from E(...) definitions)
+  with the upstream file.
+*/
+
+if (typeof AggregateError === 'undefined') {
+  globalThis.AggregateError = require('aggregate-error')
```

Comment:
Do not install AggregateError as a global

Reply:
👍